service/storage_service.cc

```diff
@@ -2476,7 +2479,8 @@ future<> storage_service::removenode(locator::host_id host_id, std::list<locator
                 try {
                     // Step 3: Start heartbeat updater
                     auto heartbeat_updater_done = make_lw_shared<bool>(false);
-                    auto heartbeat_updater = ss.node_ops_cmd_heartbeat_updater(node_ops_cmd::removenode_heartbeat, uuid, nodes, heartbeat_updater_done);
+                    auto hb_interval = std::chrono::seconds(ss._db.local().get_config().nodeops_heartbeat_interval_seconds());
```

Comment:
We get hb_interval in multiple places. It will be simpler we get the interval inside node_ops_cmd_heartbeat_updater directly.

Reply:
Right.